docs/core/compatibility/core-libraries/6.0/static-virtual-interface-methods.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Breaking change: Static virtual methods in interfaces"
+description: Learn about the .NET 6.0 breaking change where `static` interface methods can now be marked `virtual`.
+ms.date: 09/08/2021
+---
+# Static virtual methods in interfaces
+
+.NET 6 is previewing a new feature where `static` interface methods can be marked as `virtual`. This feature involves several changes to the ECMA spec to allow intermediate language (IL) metadata patterns that were previously considered illegal. For more information, see [dotnet/runtime#49558](https://github.com/dotnet/runtime/issues/49558).
+
+> [!NOTE]
+> The feature is in preview, and it's possible that additional changes or support may be introduced between .NET 6 and a future version of .NET when the feature stabilizes.
+
+## Old behavior
+
+Static interface methods could not be marked as `virtual` and would be considered illegal IL.
+
+## New behavior
+
+Static interface methods can be marked as `virtual`. The implementation of these methods is provided by types that implement the interface.
```

Comment:
Should we note that you have to enable preview features for your project?

Reply:
Same here:
```suggestion
Static interface methods can be marked as `abstract`. The implementation of these methods is provided by types that implement the interface.
```

Reply:
(and same attached comment about language vs metadata: https://github.com/dotnet/docs/pull/26013#discussion_r705470837)